lib/read_table_A.pl

```diff
@@ -1103,19 +1081,6 @@ sub read_table_A {
             $code .= "use Philips_Hue;\n";
         }
     }
-    elsif ($type eq "PHILIPS_LUX"){
```

Comment:
Hey Timothy,

could you please update the pull request so that the PHILIPS_LUX support is not removed?

Thank you,
 Lieven.